apps/gitness/src/routes.tsx

```diff
@@ -1053,6 +1045,15 @@ export const routes: CustomRouteObject[] = [
       }
     ]
   },
+  {
+    path: 'create',
+    element: (
+      <AppProvider>
```

Comment:
I didn't understand why we need to move this out of the `/` wrapper which already rendered AppProvider.

Reply:
We moved this out of the / wrapper because we don’t need the AppShell to be rendered with the CreateProject page. Keeping it inside would result in rendering a lot of unnecessary that are not needed for this specific page.